CONTRIBUTING.md

```diff
@@ -19,3 +20,8 @@ find . -name "*.md" | while read -r file; do
     npx markdown-toc $file -i
 done
 ```
+
+# Using Kubelinter
+
+Please consider using [Kubelinter](https://docs.kubelinter.io/#/) locally before submitting a PR to this repository.
+Konflux-ci's CI tests consist of Kubelinter, and it will fail the tests for PRs which are not passing Kubelinter's check.
```

Comment:
please explain how to do so

Reply:
https://docs.kubelinter.io/#/?id=usage

Would probably be more interesting to add information on how to run it recursively on a root dir